synchronizer/synchronizer.go

```diff
@@ -813,7 +814,8 @@ func (s *ClientSynchronizer) processTrustedBatch(trustedBatch *pb.GetBatchRespon
 	}
 
 	log.Debugf("resetting trusted state from batch %v", trustedBatch.BatchNumber)
-	if err := s.state.ResetTrustedState(s.ctx, trustedBatch.BatchNumber-1, dbTx); err != nil {
+	previousBatchNumber := trustedBatch.BatchNumber - 1
```

Comment:
```suggestion
```

Reply:
@cool-develope the creation of the variable `previousBatchNumber` was intended, this helps the code to express the `-1` was a thought decision instead of something else, this helps with future maintenance and the code health.